tests/core/system-snap-refresh/task.yaml

```diff
@@ -46,22 +46,10 @@ execute: |
         fi
 
         # install new target snap
-
-        snap install --dangerous "/var/lib/snapd/snaps/${TARGET_SNAP_NAME}_${currRev}.snap" &> refresh.log
-        MATCH "snapd is about to reboot the system" < refresh.log
+        snap install --dangerous --no-wait "/var/lib/snapd/snaps/${TARGET_SNAP_NAME}_${currRev}.snap"
 
         # Detect in the logs when the reboot can been triggered
-        for _ in $(seq 50); do
-            if "$TESTSTOOLS"/journal-state match-log "Waiting for system reboot"; then
-                 break
-            fi
-            sleep 2
-        done
-
-        if ! "$TESTSTOOLS"/journal-state match-log "Waiting for system reboot"; then
-            echo "Taking too long to reach waiting for reboot"
-            exit 1
-        fi
+        "$TESTSTOOLS"/journal-state match-log -n 50 --wait 2 "Waiting for system reboot"
```

Comment:
Should we have more retries (wait longer in total) now since we are not waiting for install to finish anymore?

Reply:
I have tested many times and I could not reproduce the error anymore with this time, I could add more but perhaps we can make it in case it fails

Reply:
Yeah, let's leave it as is and bump if needed. Thanks